.gitignore

```diff
@@ -118,6 +118,9 @@ tools/windows/install-help/cal/packages/
 
 # ebpf object files
 pkg/ebpf/c/*.o
+pkg/ebpf/c/*.bc
+pkg/security/ebpf/c/*.o
+pkg/security/ebpf/c/*.bc
```

Comment:
Should we just ignore `*.o` and `*.bc` files ?

Reply:
Probably 😄